include/cadmium/celldevs/delayer/transport.hpp

```diff
@@ -0,0 +1,53 @@
+/**
+* Copyright (c) 2020, Román Cárdenas Rodríguez
+* ARSLab - Carleton University
+* GreenLSI - Polytechnic University of Madrid
+* All rights reserved.
+*
+* Cell-DEVS transport output delayer.
+*/
+
+#ifndef CADMIUM_CELLDEVS_TRANSPORT_HPP
+#define CADMIUM_CELLDEVS_TRANSPORT_HPP
+
+#include <functional>
+#include <limits>
+#include <queue>
+#include <tuple>
+#include <vector>
+#include <cadmium/celldevs/delayer/delayer.hpp>
+
+
+namespace cadmium::celldevs {
+    /**
+     * @brief Cell-DEVS transport output delayer.
+     * @tparam T the type used for representing time in a simulation.
+     * @tparam S the type used for representing a cell state.
+     * @see delayer/delayer.hpp
+     */
+    template <typename T, typename S>
+    class transport_delayer : public delayer<T, S> {
+    private:
+        /// Queue with pairs <scheduled time, state to transmit>. Messages are sorted depending on the scheduled time
+        std::priority_queue<std::pair<T, S>, std::vector<std::pair<T, S>>, std::greater<std::pair<T, S>>> delayed_outputs;
+    public:
+        transport_delayer() : delayer<T, S>() {}
+
+        /// Pushes new state to the priority queue
```

Comment:
I'm starting to think you using /// with a purpose, why?

Reply:
Explained above